### DISCUSSION TITLE
Delete cookies _ga*

### DISCUSSION THREAD

**Discussion by mrazekpixon** [orestbida/cookieconsent#148] [Q&A] [answered]
Hello, first of all I would like to thank you for your work!

How do I set cookie_table: col1, for example col1: '_ga *', to delete cookies when rejecting coockies, eg _ga_EVX2X34ERCC. Thank you very much.
![cookie](https://user-images.githubusercontent.com/84787631/145946313-b190646c-7a67-41ea-97f7-f7199e995508.png)

**Comment by orestbida** [comment, marked as answer]
Hi @mrazekpixon,

you need to write a valid regex string. This is how you can match all cookies starting with `_ga`:
```javascript
{
    col1: '^_ga',
    col2: 'description ...',
    is_regex: true
},
```
